src/BizHawk.Client.Common/lua/LuaHelperLibs/GenesisLuaLibrary.cs

```diff
@@ -62,5 +67,19 @@ public void SetLayerBgW(bool value)
 			s.DrawBGW = value;
 			Settings = s;
 		}
+
+		[LuaMethodExample("genesis.add_deepfreeze_value( 0xFF00, 0x01 );")]
+		[LuaMethod("add_deepfreeze_value", "Adds an address to deepfreeze to a given value. The value will not change at any point during emulation.")]
+		public int AddDeepFreezeValue(int address, byte value)
+		{
+			return gpgx.AddDeepFreezeValue(address, value);
+		}
+
+		[LuaMethodExample("genesis.clear_deepfreeze_list();")]
+		[LuaMethod("clear_deepfreeze_list", "Clears the list of deep frozen variables")]
+		public void ClearDeepFreezeList()
+		{
+			gpgx.ClearDeepFreezeList();
+		}

```

Comment:
Is it possible to allow removing freeze from individual frozen addresses? Also maybe a getter for entry count?

Reply:
yes, absolutely, although my rudimentary knowledge of C# is the limiting factor for me here.

Reply:
While testing I got a couple more ideas. Function that returns current frozen addresses as a table (with frozen values), and more importantly ability to freeze the value that's currently in the cell without having to set it manually. I'll see what I can do...